### PR TITLE
refactoring BioCharacteristics

### PR DESCRIPTION
**Pull request by mbaudis**
Following discussions during a meeting at UCSC, this PR changes the BioCharacteristics object structure by removing one object level and changing assignment to phenotypes, diseases ... to a label.

### CODE REVIEW

**Comment by mbaudis** [comment]
@david4096 @kozbo Please check - this addresses the structural simplification for the ontology wrappers.

File: src/main/proto/ga4gh/bio_metadata.proto [hunk 1]
```diff
@@ -134,7 +140,7 @@ message BioCharacteristic {
   //     termId: "UBERON:0010033",
   //     term: "posterior part of tongue",
   //
-  repeated OntologyTerm ontologyTerms = 2;
+  repeated OntologyTerm ontologyTerms = 3;
```
Comment (kozbo) [inline src/main/proto/ga4gh/bio_metadata.proto]: We shouldn't renumber pre-existing fields in protobuf. Add a new number instead.

File: src/main/proto/ga4gh/bio_metadata.proto [hunk 2]
```diff
@@ -155,19 +161,6 @@ message BioCharacteristic {
...
-// with a first class BioMetadata object (Individual, BioSample).
-message Characteristics {
-  // Diseases associated with the parent BioMetadata object (Individual,
-  // BioSample).
-  repeated BioCharacteristic diseases = 1;
-
-  // Phenotypes associated with the parent BioMetadata object (Individual,
-  // BioSample).
-  repeated BioCharacteristic phenotypes = 2;
+  repeated OntologyTerm negatedOntologyTerms = 4;
```
Comment (kozbo) [inline src/main/proto/ga4gh/bio_metadata.proto]: this looks like a simple re-organization of existing fields, so we can keep the original protobuf number of 3

**Comment by mbaudis** [comment]
@kozbo I know; but this is still in a branch & has never seen master => ?!

File: src/main/proto/ga4gh/bio_metadata.proto (hunk 2 above)
Comment (mbaudis) [inline src/main/proto/ga4gh/bio_metadata.proto]: Can you just adjust it to what you see appropriate? Thanks!

Comment (mcourtot) [inline src/main/proto/ga4gh/bio_metadata.proto]: Small comment on 'characteristics_class': I would avoid 'class' and use maybe 'scope' to match the documentation. I would also suggest to replace 'typical examples could be' with a stronger requirement, such as a list of acceptable values (maybe with a definition, for example in the above I'm not sure of the difference between a phenotype and an observation - does it matter?)

**Comment by mbaudis** [comment]
@mcourtot Thanks for the comment; I did change the attribute name before, now to `scope`.

**Comment by mbaudis** [comment]
Consolidating this into metadata-integration as the staging branch.

**Pull request merged**